sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/TransportConsumer.cs

```diff
@@ -36,14 +35,14 @@ internal abstract class TransportConsumer
         public FaultTolerantAmqpObject<ReceivingAmqpLink> ReceiveLink { get; protected set; }
 
         /// <summary>
-        ///
+        /// TODO make this generic
         /// </summary>
         public AmqpConnectionScope ConnectionScope { get; set; }
```

Comment:
This is used within RetryPolicy.RunOperation to tell if connection is Open.
Need to make this be TransportConnectionScope and have AmqpConnectionScope derive from it.

Reply:
Fixed.